web/api/mocks/data/steps.json

```diff
@@ -1,25 +1,29 @@
 [
   {
     "id": "step-1-flow-1",
-    "name": "Flow 01 Ingest Step",
+    "name": "Flow01IngestStep",
     "description": "My Step 1 description",
     "type": "ingest",
     "sourceDatabase": "",
     "targetDatabase": "staging",
     "isValid": true,
     "options": {
-      "inputFilePath": ".",
-      "inputFileType": "xml",
-      "outputCollections": "Order,Flow 01 Ingest Step,input,newCollection",
-      "outputPermissions": "rest-reader,read,rest-writer,update",
-      "outputFileType": "json",
-      "outputURIReplacement": ""
+      "fileLocations": {
```

Comment:
@alexsmr ,
"fileLocations" is a first level element, it is not under "options". https://wiki.marklogic.com/pages/viewpage.action?spaceKey=ENGINEERING&title=Steps+and+Step+Definitions

Reply:
Oh. Man... It is little confusing in https://project.marklogic.com/jira/browse/DHFPROD-2096

@srinathgit Thanks for pointing out.

Reply:
@alexsmr ,
I see some more discrepancies between https://wiki.marklogic.com/display/ENGINEERING/Steps+and+Step+Definitions and mocks:

1. "sourceDatabase" and "targetDatabase" are inside options
2. INGEST step has only "targetDatabase" option

Reply:
@srinathgit where do you see that?

Reply:
@srinathgit this PR deals only with ingest options and some step creations features. There was a bug in the step dialog. Just commited it please refresh.